prog.go

```diff
@@ -675,24 +680,37 @@ func (p *Program) testRun(opts *RunOptions) (uint32, time.Duration, error) {
 		Cpu:         opts.CPU,
 	}
 
+	if attr.Repeat == 0 {
```

Comment:
The initial comment on `RunOptions.Repeat` reads `Defaults to 1` already. Does this serve a purpose other than simplifying the break logic below?